### PR TITLE
Develop  to master

### PR DESCRIPTION
**Pull request by Dongmo12**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+}
+.menu span.caption {
+    font-style: italic;
+}
+.intro-row {
+    height: 150px;
+}
+.floating-intro {
+    margin-left: auto;
+    /* margin-right: -150px; */
```
Comment (JamesOkunlade) [inline css/style.css]: Remove this line.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+    display: inline-block;
+    position: relative;
+}
+.person-of-the-week .content {
+    display: inline-block;
+    position: relative;
+}
+.frame {
+    border-radius: 10px;
+    /* background-color: black; */
```
Comment (JamesOkunlade) [inline css/style.css]: Remove this line too. Don't leave unused styles in your code as comments.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hello guys!!, 

Great work on the design teardown project. 

There are some avoidable errors in your HTML code. Parse your code through the [w3C markup validator](https://validator.w3.org/) to identify and fix these errors. 

Also, the commits are too unevenly distributed for a collaborative project. Ensure to switch more often in your future collaborative projects. 

Fix the HTML errors and request another code review. Add me as the TSE so I can follow up.

Happy coding!

**Comment by ibrolive** [comment]
Hi @JamesOkunlade. Thanks for your feedback and for taking the time to review this. HTML errors have been fixed.

**Review by JamesOkunlade — APPROVED** [review]
### **Well did guys!!!**

You're good to go. Merge to master, close pull request and fill the project completion form.

Good luck with your future projects.:+1:+1:

**Pull request merged**